config/robots.rb

```diff
@@ -1988,11 +1988,16 @@
 
   # Heron LTHR 96 Pipeline
 
+  bravo_robot do
+    from 'LTHR Cherrypick', bed(1)
+    to 'LTHR RT-S', bed(5)
+  end
+
   custom_robot('bravo-lthr-96-rt-to-lthr-96-pcr-1-and-2',
                name: 'bravo LTHR RT => LTHR PCR 1 and 2',
```

Comment:
Change robot name to RT-S ?

Reply:
Ah, but it caters for both types. I guess it's fine then

Reply:
It's a bit frustrating that the different purposes at this stage are for our convenience, not the users.